test/scripts/integration_tests.py

```diff
@@ -122,13 +182,36 @@ def runTest(self):
         self.__check_metrics(6, 1, 3, 2, 1)
         self.__kubectl.delete(_NEMO_CUSTOM_RESOURCE_FILE, True)
         self.__check_metrics(7, 0, 3, 3, 1)
+        if self.__lostromos_process:
+            self.__lostromos_process.kill()
+
+        print("Starting Lostromos with helm controller")
+        self.__lostromos_process = subprocess.Popen(
+            [
+                _LOSTROMOS_EXE,
+                "start",
+                "--config",
+                _TEST_DATA_DIRECTORY + "/helm/wait-config.yaml",
+                "--helm-tiller",
+                self.__minikube_ip + ":32664",
+            ],
+        )
+        print("Started Lostromos with PID: {}".format(self.__lostromos_process.pid))
+        sleep(15)
+        #self.__wait_for_tiller_start()
```

Comment:
Not sure if this is ready for final review yet, but uncommenting/using this line looks like the only issue I'm seeing.